### ISSUE TITLE
CLI appears to require ruby 2.1

### ISSUE THREAD

**Issue by mikeyyuen** [kontena/kontena#1266]
When running

```
$ DEBUG=true kontena cloud login
... after login from browser ...
  * Parsing params: "code=<omitted>"
/Library/Ruby/Gems/2.0.0/gems/kontena-cli-0.16.1/lib/kontena/cli/localhost_web_server.rb:85:in `serve_one': undefined method `to_h' for [["code", "<omitted>"]]:Array (NoMethodError)
	from /Library/Ruby/Gems/2.0.0/gems/kontena-cli-0.16.1/lib/kontena/cli/cloud/login_command.rb:85:in `block in execute'
```

it seems that to_h is a 2.1 feature

http://stackoverflow.com/questions/31683993/undefined-method-to-h-on-ruby-array

However the doc's say 2.0 should work

https://www.kontena.io/docs/getting-started/quick-start

**Comment by kke** [comment]
Well spotted!

**Comment by jakolehm** [comment]
Fixed in #1267
